spec/javascript/components/applications/application_form/PaperApplicationForm.test.js

```diff
@@ -2,15 +2,17 @@
 import React from 'react'
 import { clone } from 'lodash'
 import { mount } from 'enzyme'
+import { act } from 'react-dom/test-utils'
```

Comment:
This diff is hard to read but I did several things:
1. Moved "Language" test up to the top to match how the actual form is arranged
1. Changed the way we select demographics and simulate blur to be consistent with supp app page unit tests and remove lines of code and copy pasta
1. Removed wait(100)s by replacing them with awaits, acts, and wrapper.update()s
1. Added a test for alt contact validation